### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by long1250** [baisaganov/binance-nft#6]
Can you add a click delay to it?
For example, the click time is 5ms
Otherwise, you may not be able to pass binance's anti-robot #

**Comment by COX888** [comment]
Да, сегодня посыпались ошибки, когда запустил скрипт перед торгами

**Comment by bogdants451** [comment]
Ошибки посыпались, со всех устройств что я пробовал. Во первых CORB гугловский рестартит скрипт. Прикреплю фото ошибок.
![Uploading 20211130_150538.jpg…]()

**Comment by bogdants451** [comment]
![Uploading 20211130_150538.jpg…]()
